src/Main.java

```diff
@@ -1,5 +1,57 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        System.out.println("Задача 1");
+        int [] expenses = {500, 2000, 300, 1500, 800};
+        int sum = 0;
+        int sum1 = 0;
+
+        for (int i = 0; i < expenses.length; i++) {
+            sum += expenses[i];
+        }
+        System.out.println("Сумма трат за месяц составила " + sum + " рублей");
+
+        for (int element : expenses) {
+            sum1 += element;
+        }
+        System.out.println("Сумма трат за месяц составила " + sum1 + " рублей");
+        System.out.println();
+
+        System.out.println("Задача 2");
+        int maxExpenses = -1;
+        int minExpenses = 2500;
+
+        for (int i = 0; i < expenses.length; i++) {
+            if (expenses[i] > maxExpenses) {
+                maxExpenses = expenses[i];
+            }
+        }
+        System.out.println("Максимальная сумма трат за неделю составила " + maxExpenses + " рублей");
+
+        for (int i = 0; i < expenses.length; i++) {
+            if (expenses[i] < minExpenses) {
+                minExpenses = expenses[i];
+            }
+        }
+        System.out.println("Максимальная сумма трат за неделю составила " + minExpenses + " рублей");
+        System.out.println();
+
+        System.out.println("Задача 3");
+        int sum3 = 0;
+        for (int i = 0; i < expenses.length; i++) {
+            sum3 += expenses[i];
+        }
+        float averageExpenses = sum3/4;
```

Comment:
1) Чтобы получить среднее надо сумму поделить на количество элементов. Элементов в массиве не 4, а 5
2) В массиве может быть либо 4 либо 5 элементов, поэтому использовать число не получится. Нужно количество брать из свойств массива
```suggestion
        float averageExpenses = sum3/expenses.length;
```